upload/tasks.py

```diff
@@ -57,17 +56,21 @@ def get_or_create_package(pid_v3, user_id):
     return task_get_or_create_package(pid_v3, user_id)
 
 
-@celery_app.task(name='Validate article and issue data')
-def task_validate_article_and_issue_data(file_path, package_id, issue_id):
+@celery_app.task(bind=True, name='Validate article and issue data')
+def task_validate_article_and_issue_data(self, file_path, package_id, issue_id):
     task_validate_article_and_journal_issue_compatibility.apply_async(kwargs={
         'package_id': package_id,
         'file_path': file_path,
         'issue_id': issue_id,
     })
-    task_validate_article_is_unpublished.apply_async(kwargs={
-        'package_id': package_id,
-        'file_path': file_path,
-    })
+    try:
+        task_validate_article_is_unpublished.apply_async(kwargs={
```

Comment:
para que serve?

Reply:
@robertatakenaka Isso é a task que verifica se um artigo é inédito.